docs/HowToFAQ/Errors-Rewriter.md

```diff
@@ -153,7 +153,7 @@ the same indentation. This warning occurs when the observed indentation is likel
 a misunderstanding of the code, particularly of the precedence of operations. It is suggested
 to use parentheses or to redo the indentation to make the structure of the expression clearer.
 
-<!-- FILE ./DafnyCore/Resolver/RunAllTestsMainMethod.cs -->
+<!-- FILE ./DafnyCore/Rewriters/RunAllTestsMainMethod.cs -->
```

Comment:
Just curious: what do these links do? Are they just notes for us pointing to where we emit the errors in the source code?